WooCommerce/Classes/POS/Presentation/Settings/PointOfSaleSettingsHelpDetailView.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
+
+struct PointOfSaleSettingsHelpDetailView: View {
+    @Environment(\.dismiss) private var dismiss
+
+    var body: some View {
+        NavigationStack {
+            VStack(alignment: .leading) {
+                Text("Help Settings")
+                    .font(.title2)
+                Text("Help-related configuration")
+                    .font(.caption)
+                    .foregroundStyle(.secondary)
+            }
+            .padding()
+            .toolbar {
```

Comment:
Here's a `NavigationStack` gotcha: I had to add the `.toolbar` for the dismiss action to all destination views, rather than once to the parent Settings view, if we do, then these destinations will be presented full-screen as part of the stack, rather than encapsulated to the right-side only.